www/admin/javascript/jquery-1.8.3.min.js

```diff
@@ -1,2 +1,5562 @@
\ No newline at end of file
+/*! jQuery v1.8.3 jquery.com | jquery.org/license */
```

Comment:
This file should be excluded from autoformat. I'm not sure why it's part of the repo at all.